admin/scripts/modules/uvod.php

```diff
@@ -93,7 +93,7 @@
 }
 
 if(post('zmenitUdaj')) {
-  dbUpdate('uzivatele_hodnoty', post('udaj'), ['id_uzivatele' => $uPracovni->id()]);
+  $uPracovni->zmenitUdaj(post('udaj'));
```

Comment:
Tady se asi nevyhnem tomu udělat na čtení / nastavení občanky metodu, která bude to šifrování spravovat. Obecně ten nápad mít metodu na úpravu "jakože sloupce" a v ní ošetřit, jestli jdou přímo uložit data, nebo je potřeba to nějak ošetřit, mi vlastně přijde docela zajímavé řešení. Na druhý pohled mi to ale [smrdí CQRS](https://www.martinfowler.com/bliki/CQRS.html) když vidím, že se to separátně dešifruje, a pokud bychom chtěli tenhle problém probourat a jít cestou dynamických jazyků, asi by bylo ultimátním řešením to udělat [jak to má Python](https://stackoverflow.com/questions/6618002/python-property-versus-getters-and-setters), což by i v php teoreticky šlo (byť to už jsme vyběhli značne mimo téma úkolu).

V modelu ještě prosím bacha na používaný jquery styl pro gettery / settery (a kontrolu, jestli už tam getovací metoda není). Na tomto místě to hold bude potřeba oifovat a občanku ukládat jinak než zbytek atributů. Dtto getování o nějaké řádky níž.